kuma/users/migrations/0019_auto_20201105_0411.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.16 on 2020-11-05 04:11
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0018_user_subscriber_number"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="user",
+            name="mozillians_url",
+        ),
+        migrations.AddField(
+            model_name="user",
+            name="pmo_url",
+            field=models.TextField(
+                blank=True,
+                validators=[
+                    django.core.validators.RegexValidator(
+                        "^https?://people\\.mozilla\\.org/u/",
```

Comment:
It seems that the Mozilla People Directory uses `/p/...` rather than `/u/...`, but I don't think we should care about anything past the domain, so I'd suggest:
```suggestion
                        "^https?://people\\.mozilla\\.org/",
```